plugins/mission/mission_item.cpp

```diff
@@ -117,4 +131,59 @@ std::string MissionItem::to_str(MissionItem::CameraAction camera_action)
     }
 }
 
+bool operator==(const MissionItem &lhs, const MissionItem &rhs)
+{
+    return (lhs.get_latitude_deg() == rhs.get_latitude_deg()
+            || isnan(lhs.get_latitude_deg()) && isnan(rhs.get_latitude_deg()))
```

Comment:
I think it would be better to make it explicit what the operator order is.

Reply:
Oh, right. macOS is warning about that, and apparently I did not fix it on the right computer. I'll change it!

Reply:
Haha, it's a warning. The operator precedence is clear in theory but it's nice for the reader to make it explicit in my opinion.